src/app/snapp_test_transaction/snapp_test_transaction.ml

```diff
@@ -7,6 +7,14 @@ let constraint_constants = Genesis_constants.Constraint_constants.compiled
 
 let proof_level = Genesis_constants.Proof_level.Full
 
+(* changes first char to lower case unlike Mina_graphql.Reflection.underToCamel*)
+let underToCamel s =
```

Comment:
would it make sense to call the one in `Mina_graphql`, then change just the first character, rather than duplicate most of the code?